app/src/main/java/com/yello/presentation/main/look/LookFragment.kt

```diff
@@ -0,0 +1,13 @@
+package com.yello.presentation.main.look
+
+import com.example.ui.base.BindingFragment
+import com.yello.R
+import com.yello.databinding.FragmentLookBinding
+
+class LookFragment : BindingFragment<FragmentLookBinding>(R.layout.fragment_look) {
+
+    companion object {
+        @JvmStatic
+        fun newInstance() = LookFragment()
+    }
```

Comment:
<메모> 번들로 받을 인자들을 보내는 뷰 대신, 여기 뷰에서 번들로 묶어서 받을 수 있는 방법 ! 
채연이 예시 :
```suggestion
        @JvmStatic
        fun newInstance(index: Int, memberId: Int) = MemberStoryFragment().apply {
            val args = Bundle()
            args.putInt(ARGS_INDEX_STORY, index)
            args.putInt(ARGS_MEMBER_ID, memberId)
            arguments = args
        }
```

Reply:
굿굿티비 ^___^